Description: Fix for a lintian spelling error in /usr/bin/gbemol
Author: Andrea Colangelo <warp10@ubuntu.com>

--- gbemol-0.3.2.orig/src/gbemol-preferences.c
+++ gbemol-0.3.2/src/gbemol-preferences.c
@@ -234,7 +234,7 @@ gbemol_preferences_new (GbemolApp* app)
 	obj->priv->chk_update = gtk_check_button_new_with_label (_("Update library on start?"));
 	gtk_misc_set_alignment (GTK_MISC (lbl), 0.0, 0.5);
 	gtk_tooltips_set_tip (obj->priv->tips, obj->priv->chk_update, 
-			_("Wheter the music library should be updated when the program is started"), NULL);
+			_("Whether the music library should be updated when the program is started"), NULL);
 	gtk_table_attach_defaults (GTK_TABLE (table), obj->priv->chk_update, 0, 1, 6, 7);
 
 	/**** Audio Tab ****/
